src/main/java/com/homework/lesson14/abstractfactory/RuFerrariImpl.java

```diff
@@ -0,0 +1,8 @@
+package com.homework.lesson14.abstractfactory;
+
+class RuFerrariImpl implements Ferrari {
+    @Override
+    public long getFerrariPrice() {
+        return 30000;
```

Comment:
Is better to use constant for this number.
Example:
`private static final int MAX_SPEED=3000;`